core/services/versionchooser/utils/chooser.py

```diff
@@ -144,8 +148,11 @@ async def get_bootstrap_version(self) -> str:
         Returns:
             str: The image version string for the bootstrap container.
         """
-        bootstrap = await self.client.containers.get(self.bootstrap_name)  # type: ignore
-        return str(bootstrap["Config"]["Image"])
+        try:
+            bootstrap = await self.client.containers.get(self.bootstrap_name)  # type: ignore
+            return str(bootstrap["Config"]["Image"])
+        except Exception as e:
+            return "Unknown"
```

Comment:
Can't we just f-string the error here ?
Besides that, we should log the error as well.

Reply:
f-string-ing the error could cause a huge text to show up in the sidebar, that's why I opted not to...
I'll add logging